Tools/samples.h

```diff
@@ -39,11 +39,11 @@ namespace AnaSamples
 
     double getWeight() const {return weight_;}
     const std::vector<std::string>& getFilelist() const {return filelist_;}
-    template<class T> void addFilesToChain(T* chain,  int startfile =0, int filerun= -1) const
+    template<class T> void addFilesToChain(T* chain, int startfile=0, int filerun=-1) const
     {
       if(filelist_.size() == 0) readFileList();
       if(filerun<0)filerun=filelist_.size();
-      for(int fn = startfile; fn < startfile+filerun && fn<filelist_.size(); fn++)
+      for(unsigned int fn = startfile; int(fn) < startfile+filerun && fn<filelist_.size(); fn++)
```

Comment:
please use c++ style typecast

Reply:
Ok. I was conflicted because making a new int might be less work than a static_cast.

Reply:
that is doing a static cast